nilearn/plotting/image/img_plotting.py

```diff
@@ -30,29 +30,30 @@
 )
 from nilearn._utils.extmath import fast_abs_percentile
 from nilearn._utils.logger import find_stack_level
-from nilearn._utils.ndimage import get_border_data
 from nilearn._utils.niimg import safe_get_data
 from nilearn._utils.param_validation import check_params, check_threshold
-from nilearn.datasets import load_mni152_template
 from nilearn.image import (
     get_data,
     iter_img,
     math_img,
     new_img_like,
     resample_to_img,
 )
-from nilearn.image.resampling import reorder_img
 from nilearn.maskers import NiftiMasker
 from nilearn.masking import apply_mask, compute_epi_mask
 from nilearn.plotting import cm
 from nilearn.plotting._utils import (
     check_threshold_not_negative,
     create_colormap_from_lut,
-    get_cbar_ticks,
     get_colorbar_and_data_ranges,
     save_figure_if_needed,
 )
 from nilearn.plotting.displays import get_projector, get_slicer
+from nilearn.plotting.image.utils import (
+    MNI152TEMPLATE,
+    get_cropped_cbar_ticks,
```

Comment:
This one need not necessarily be public. However as it is a utility function, I wanted to take it out. 
Is it preferable to put it to another module `_utils.py` ?

Reply:
maybe in nilearn/plotting/_utils.py ?

Reply:
As it is related to image processing, I wanted to keep it under `plotting.image`.